speed_games/results_tool/styles/index.css

```diff
@@ -0,0 +1,229 @@
+/* *
+* @license Copyright 2023 Google Inc. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* @author ghys@google.com (Lou Ghys) */
+
+body {
+    background: url("assets/space-background-pink.png");
```

Comment:
The tool should be run inside the speed_games/results_tools as root, otherwise it doesn't work correctly.
The path is correct when run from the correct location